### PR TITLE
feat: add prettier to the project

### PR DESCRIPTION
**Pull request by MaciekPon**
- add prettier configuration file

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "tabWidth": 2,
+  "semi": true,
+  "printWidth": 80,
+  "singleQuote": false,
+  "arrowParens": "always",
+  "bracketSpacing": true
```
Comment (ruljin) [inline .prettierrc:7]: Add `trailingComma`

**Review by ruljin — CHANGES_REQUESTED** [review]
Changes requested

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "tabWidth": 2,
+  "semi": true,
+  "printWidth": 80,
+  "singleQuote": false,
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "trailingComma": "es5"
```
Comment (ruljin) [inline .prettierrc:8]: Change to`all`

**Review by ruljin — CHANGES_REQUESTED** [review]
Changes requested

**Comment by alicjaguzik** [comment]
LGTM

**Review by JustD1d1t — APPROVED** [review]
LGTM

**Review by ruljin — APPROVED** [review]

**Pull request merged**